src/content/docs/learn/window-menu.mdx

```diff
@@ -266,6 +281,12 @@ fn main() {
 
             app.set_menu(menu)?;
 
+            let menu_image_update = Image::from_bytes(include_bytes!("../icons/menu_update.png")).unwrap();
+            // You can also update the submenu icon dynamically
+            file_menu.set_icon(Some(menu_image_update))?;
+            // Or set a native icon
+            file_menu.set_native_icon(Some(tauri::menu::NativeIcon::Folder))?;
```

Comment:
We should note that only one of them applies? (could use a platform config to use native icon on macOS and `set_icon` on others)